Add Query Syntax container (#390)
risingwavelabs/risingwave-docs: cloud/update-database-version.mdx

---
title: "Update RisingWave version"
description: "When a newer version of RisingWave becomes available, you can easily update your project to the latest version. Check out the [Release Notes of RisingWave](/changelog/release-notes) to learn about the exciting new features in each version."
---

## What to expect during upgrade

> Before starting the upgrade, here's what you should know:

Reviewer: ```suggestion
Here’s what you should know about the upgrade:
```

* RisingWave Cloud automatically backs up your data before the upgrade begins
* During the upgrade process:
  * Project operations will be temporarily unavailable
  * Streaming pipelines will be halted and experience a temporary downtime
  * Query Serving, being more lightweight, will typically recover faster than streaming pipelines

>   * You won't be charged for compute resources during this period

Reviewer: ```suggestion
**Before the upgrade starts**:

- RisingWave Cloud automatically backs up your data.

**During the upgrade process**:

- Project operations will be temporarily unavailable.

- Streaming pipelines will be halted and experience temporary downtime.

- Query Serving, being more lightweight, will typically recover faster than streaming pipelines.

- You won’t be charged for compute resources during this period.
```
